components/tidb_query_expr/src/impl_json.rs

```diff
@@ -156,6 +156,47 @@ pub fn json_merge(args: &[Option<JsonRef>]) -> Result<Option<Json>> {
     Ok(Some(Json::merge(jsons)?))
 }
 
+#[rpn_fn(nullable, raw_varg)]
```

Comment:
Maybe we can remove `nullable` and add `writer` here?

Reply:
We don't need `raw_varg` for `JSON_QUOTE` as it only accept a `string` input: https://dev.mysql.com/doc/refman/5.7/en/json-creation-functions.html#function_json-quote